src/human_aware_rl/human/data_processing_utils.py

```diff
@@ -0,0 +1,274 @@
+import json
+import time
+
+import numpy as np
+
+from overcooked_ai_py.agents.benchmarking import AgentEvaluator
+from overcooked_ai_py.mdp.actions import Action, Direction
+from overcooked_ai_py.mdp.overcooked_mdp import (
+    ObjectState,
+    OvercookedGridworld,
+    OvercookedState,
+    PlayerState,
+)
+
+AI_ID = "I am robot, bite my shiny metal ass!"
```

Comment:
From the looks of it, it was used to convert human data in the old schema to the current schema and this is just an ID needed in the conversion. Not sure how much it is still used since existing codes mostly work off pickled dataframes directly.